pages/bib/[id].tsx

```diff
@@ -0,0 +1,59 @@
+import Head from "next/head"
+import { Heading } from "@nypl/design-system-react-components"
+
+import Layout from "../../src/components/Layout/Layout"
+import { PATHS, SITE_NAME } from "../../src/config/constants"
+import { fetchBib } from "../api/bib"
+import { mapQueryToBibParams } from "../../src/utils/bibUtils"
+
+/**
+ * The Bib page is responsible for fetching and displaying a single Bib's details.
+ */
+export default function Bib({ bib, annotatedMarc }) {
+  console.log(bib)
```

Comment:
Should these console.logs be removed?

Reply:
They'll be removed eventually, I just added them to bypass the unused vars warning for now.